frontend/src/__tests__/unit-test/signup/signup.test.tsx

```diff
@@ -7,10 +7,30 @@ import {
   EMAIL_EMPTY_ERRORMESSAGE,
   INVALID_CONFIRMPWD_ERRORMESSAGE,
   INVALID_PWD_ERRORMESSAGE,
-  INVALID_USERNAME_ERRORMESSAGE,
+  NAME_EMPTY_ERRORMESSAGE,
   PASSWORD_EMPTY_ERRORMESSAGE,
   USERNAME_EMPTY_ERRORMESSAGE,
 } from '@/constants/images';
+import { toast } from 'react-toastify';
+
+interface sucessdata {
+  status: number;
```

Comment:
I see this used multiple places, can we have a seperate types folder? and make it reusable